### PR TITLE
PIM-4261: last fixes

### PR DESCRIPTION
**Pull request by jjanvier**

### CODE REVIEW

File: spec/Pim/Bundle/CatalogBundle/MongoDB/Normalizer/FileNormalizerSpec.php [hunk 1]
```diff
@@ -24,9 +24,14 @@ function it_normalizes_media(FileInterface $file)
...
         $file->getOriginalFilename()->willReturn('myfile.pdf');
+        $file->getId()->willReturn(152);
 
         $this
             ->normalize($file, 'mongodb_json', [])
-            ->shouldReturn(['filename' => 'key/of/the/file.pdf', 'originalFilename' => 'myfile.pdf']);
+            ->shouldReturn([
+                'id' => 152,
+                'key' => 'key/of/the/file.pdf',
+                'originalFilename' => 'myfile.pdf',
```
Comment (phaseinducer) [inline spec/Pim/Bundle/CatalogBundle/MongoDB/Normalizer/FileNormalizerSpec.php]: align please

File: src/Pim/Bundle/BaseConnectorBundle/Writer/DirectToDB/MongoDB/ProductWriter.php [hunk 2]
```diff
@@ -28,6 +27,8 @@
  * @author    Benoit Jacquemont <benoit@akeneo.com>
  * @copyright 2014 Akeneo SAS (http://www.akeneo.com)
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ *
+ * Rewrite this so that it work without the MediaManager
```
Comment (phaseinducer) [inline src/Pim/Bundle/BaseConnectorBundle/Writer/DirectToDB/MongoDB/ProductWriter.php]: "to make it work"

File: src/Pim/Bundle/InstallerBundle/FixtureLoader/ConfigurationRegistry.php [hunk 3]
```diff
@@ -230,8 +229,8 @@ protected function getConfiguration()
      */
     protected function parseConfiguration(ConfigCache $configCache)
     {
-        $config = array();
-        $resources = array();
+        $config = [];
+        $resources = [];
```
Comment (phaseinducer) [inline src/Pim/Bundle/InstallerBundle/FixtureLoader/ConfigurationRegistry.php]: align please

File: src/Pim/Component/Catalog/Comparator/Attribute/FileComparator.php [hunk 4]
```diff
@@ -46,8 +46,9 @@ public function compare($data, $originals)
         $default = ['locale' => null, 'scope' => null, 'data' => ['filePath' => null]];
         $originals = array_merge($default, $originals);
 
-        if (isset($data['data']['filePath'])
-            && isset($originals['data']['filePath']) && $data['data']['filePath'] === $originals['data']['filePath']) {
+        if (isset($data['data']['filePath']) &&
+            isset($originals['data']['filePath']) &&
+            $data['data']['filePath'] === $originals['data']['filePath']) {
```
Comment (phaseinducer) [inline src/Pim/Component/Catalog/Comparator/Attribute/FileComparator.php]: `) {` should be on a new line

**Comment by nidup** [comment]
\o/ :fire: :fire: :fire: :facepunch: :wolf: :wolf: :wolf:

File: src/Pim/Bundle/BaseConnectorBundle/Writer/DirectToDB/MongoDB/ProductWriter.php (hunk 2 above)
Comment (jjanvier) [inline src/Pim/Bundle/BaseConnectorBundle/Writer/DirectToDB/MongoDB/ProductWriter.php]: done

File: spec/Pim/Bundle/CatalogBundle/MongoDB/Normalizer/FileNormalizerSpec.php (hunk 1 above)
Comment (jjanvier) [inline spec/Pim/Bundle/CatalogBundle/MongoDB/Normalizer/FileNormalizerSpec.php]: done

File: src/Pim/Bundle/InstallerBundle/FixtureLoader/ConfigurationRegistry.php (hunk 3 above)
Comment (jjanvier) [inline src/Pim/Bundle/InstallerBundle/FixtureLoader/ConfigurationRegistry.php]: done

File: src/Pim/Component/Catalog/Comparator/Attribute/FileComparator.php (hunk 4 above)
Comment (jjanvier) [inline src/Pim/Component/Catalog/Comparator/Attribute/FileComparator.php]: done

**Pull request merged**